### PR TITLE
added --list-tenants internal command + unit test

### PR DESCRIPTION
**Pull request by deepchoudhery**
- added --list-tenants <Username>
- added unit tests for list-tenants
- hid other internal commands

### CODE REVIEW

File: src/MsIdentityScaffolding/Microsoft.DotNet.MsIdentity/Tool/MsAADTool.cs [hunk 1]
```diff
@@ -125,10 +134,67 @@ private void PrintServicePrincipalList(IList<DirectoryObject> graphObjects, bool
...
+        }
+
+        internal async Task<string> PrintTenantsList()
+        {
+            string outputJsonString = string.Empty;
+            IList<TenantInformation> tenantList = new List<TenantInformation>();
+            if (AzureManagementAPI != null)
+            {
+                var tenantsJsonString = await AzureManagementAPI.ListTenantsAsync();
+                Console.WriteLine(tenantsJsonString);
```
Comment (zahalzel) [inline src/MsIdentityScaffolding/Microsoft.DotNet.MsIdentity/Tool/MsAADTool.cs:149]: Is this for testing?

File: src/MsIdentityScaffolding/Microsoft.DotNet.MsIdentity/Tool/MsAADTool.cs [hunk 2]
```diff
@@ -6,61 +6,69 @@
...
-        private void PrintApplicationsList(IList<DirectoryObject> graphObjects, bool outputJson)
+        internal async Task<string> PrintApplicationsList()
         {
+            string outputJsonString = string.Empty;
+            var graphObjects = await GraphServiceClient.Me.OwnedObjects
+                            .Request()
+                            .GetAsync();
+            var grapphObjects = graphObjects.ToList();
+            string text = JsonSerializer.Serialize(grapphObjects);
+            Console.WriteLine(text);
```
Comment (zahalzel) [inline src/MsIdentityScaffolding/Microsoft.DotNet.MsIdentity/Tool/MsAADTool.cs:69]: Testing?

**Review by zahalzel — APPROVED** [review]

File: src/MsIdentityScaffolding/Microsoft.DotNet.MsIdentity/Tool/MsAADTool.cs (hunk 2 above)
Comment (deepchoudhery) [reply src/MsIdentityScaffolding/Microsoft.DotNet.MsIdentity/Tool/MsAADTool.cs:69]: ah I thought I removed those. nice catch.

File: src/MsIdentityScaffolding/Microsoft.DotNet.MsIdentity/Tool/MsAADTool.cs (hunk 1 above)
Comment (deepchoudhery) [reply src/MsIdentityScaffolding/Microsoft.DotNet.MsIdentity/Tool/MsAADTool.cs:149]: yup. nice catch ty

**Comment by deepchoudhery** [comment]
Fixes #1533

**Pull request merged**